crates/storage_models/src/query/generics.rs

```diff
@@ -359,6 +361,37 @@ where
     .attach_printable_lazy(|| "Error filtering records by predicate")
 }
 
+#[instrument(level = "DEBUG", skip_all)]
+pub(super) async fn generic_filter_order<T, P, R, Expr>(
+    conn: &PgPooledConn,
+    predicate: P,
+    limit: Option<i64>,
+    expr: Expr,
```

Comment:
Please name this parameter as `order_by` or `order_by_clause` for easier understanding.